test/bigtest/interactors/BatchGroupConfigurationSettingsInteractor.js

```diff
@@ -4,4 +4,8 @@ import {
 
 export default @interactor class BatchGroupConfigurationSettingsInteractor {
   static defaultScope = '[data-test-batch-group-configuration-settings]';
+
+  whenLoaded() {
+    return this.timeout(5000).when(() => this.isLoaded);
```

Comment:
but there is no `isLoaded`, as far as I can see

Reply:
oops, missed it, thx